### PR TITLE
Turn all TODO items in source code into official Issues

### PR DESCRIPTION
**Pull request by damivik**
closes issue #281

### CODE REVIEW

File: astroquery/fermi/core.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 @async_to_sync
 class FermiLATClass(BaseQuery):
     """
-    TODO: document
+    # TODO: document
```
Comment (ceb8) [inline astroquery/fermi/core.py:20]: What is the reason for all the added `#` symbols?

File: docs/gaia/gaia.rst [hunk 2]
```diff
@@ -92,15 +92,47 @@ degrees around an specific point in RA/Dec coordinates.
...
+ 0.019752561500226976 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636090407832546944                     
+  0.01991656886177004 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066940132132352                     
+ 0.020149589233310516 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066871411763968                     
+ 0.020307185970548904 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089514478069888                     
+ 0.020454730686780127 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066940131244288                     
+ 0.020802655215768254 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636067141990822272                     
+ 0.021615117161838747 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636090369173963776                     
+Length = 50 rows
+
```
Comment (ceb8) [inline docs/gaia/gaia.rst:135]: I am unclear on what these changes in the Gaia docs have to do with making in code to-dos into issues. Can you clarify?

**Review by ceb8 — CHANGES_REQUESTED** [review]
It's great to have these inline to-dos as issues! Thank you! 

In addition to my inline comments, I think it would be great if you could move the "References TODO...." into the issue description and also include the relevant code snippet. I.e.
```
class GetFermilatDatafile(object):
    """
    # TODO: document
    # TODO: Fail with useful failure messages on genuine failures
    (this doesn't need to be implemented as a class)
    """
```
To help people know what they are looking for.

**Comment by ceb8** [comment]
Oh, also! Check off all the to-dos you turned into issues in https://github.com/astropy/astroquery/issues/281 so we know they've been done. :)

File: astroquery/fermi/core.py (hunk 1 above)
Comment (damivik) [reply astroquery/fermi/core.py:20]: Okay. I will the update the issue description.

Sorry about the # symbols. I misunderstood the instruction in #281: There are many instances of "TODO" in comments in the source. Each should be given its own issue (or simply resolved) and the **TODO marked with the issue #.**

File: docs/gaia/gaia.rst (hunk 2 above)
Comment (damivik) [reply docs/gaia/gaia.rst:135]: oops 😬. I was working on this #1217, I must have mixed up the branches. Sorry, I'm still learning how to use git.
I will delete the commit.

**Pull request closed without merging**